README.md

```diff
@@ -15,15 +15,15 @@ If set as a cron job - updates existing alarms if
 Read/Write Capacity Units DynamoDB table parameters changed.
 
 Usage:
-    dynamodb_create_cloudwatch_alarms [options]
-    dynamodb_create_cloudwatch_alarms [-h | --help]
+    dynamodb-create-cloudwatch-alarms (-s <sns_topic_arn>) [-r <region>] [-p <prefix>] [--debug] 
+    dynamodb-create-cloudwatch-alarms [-h | --help]
 
 Options:
-    --debug   Don't send data to AWS
-    
-Examples:
-    dynamodb_create_cloudwatch_alarms
-    dynamodb_create_cloudwatch_alarms --debug
+     -s <sns_topic_arn>    For sending alarm ( require )
```

Comment:
Can you also give a usage example so people know what to put as an sns topic arn?